prdoc/pr_5526.prdoc

```diff
@@ -0,0 +1,13 @@
+title: "Fix enact_candidate weight generation"
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR works around an issue in multivariate linear regression of weight generation.
```

Comment:
Is there an issue for this and shall we reference it from the benchmarks, so the values can be changed back once fixed?

Reply:
I'll need to dig a bit deeper to provide meaning feedback for the issue, but I don't think there's a need to change those values back once fixed. The purpose of linear regression is to find the right correlation and if it can be achieved with just a points, there is no need to add more.